install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -156,7 +156,16 @@ data:
         - mountPath: /etc/certs/
           name: istio-certs
           readOnly: true
+      {{ if .Values.global.gcpIdentityEnabled }}
```

Comment:
Maybe SDSApi is a better name for this flag? Essentially we have this change is because of the SDS, but not gcpIdentity. Sorry I did not make it clear in our last chat.

Reply:
renamed to ```sdsAPIEnabled```.